docs/curation-review-tut.md

```diff
@@ -0,0 +1,570 @@
+
+The last section describes how gene symbol conversion should be applied to each of the layers. This is similar to the
+`layer_descriptions` field above, but there are only three permitted values: `raw`, `log1p`, and `sqrt`:
+
+```
+fixup_gene_symbols:
+  X: log1p
+  raw.X: raw
+```
+
+This tells the script how each each layer was transformed from raw values that can be directly summed. `raw` means that
+the layer contains raw counts or some linear tranformation of raw counts. `log1p` means that the layer has `log(X + 1)`
+for each the raw `X` values. `sqrt` means `sqrt(X)` (this is not common). For layers produced by Seurat's normalization
+or SCTransform functions, the correct choice is usually `log1p`.
+
+**Note**: If a layer is not specified in the `fixup_gene_symbols` section, then it will not be carried over into the curated object. If no layers are specified in `fixup_gene_symbols` or if `fixup_gene_symbols` is not included the `config.yaml` file at all, then all layers will be carried over into the curated object.
+
+<br/>
+
+
+### `cellxgene schema apply`
+ 
+<br/>
+
+In order to use the `cellxgene schema apply` command, you will need to pass the following required arguments:
+- `--source-h5ad` your original `AnnData` file
+- `--remix-config` the `config.yaml` file that we specified above 
+- `--output-filename` the name of the resulting `AnnData` that is consistent with the cellxgene schema
+
+<br/>
+
+The next step will be to validate the resulting object.
+
+<br/>
+
+### `cellxgene schema validate`
+
+<br/>
+
+In order to validate the remixed object, needs to simply run `cellxgene schema validate path_to_remixed_anndata.h5ad`. If there has been no terminal output from the function, then your object has been validated successfuly and is ready for upload!
+
+<br/>
+
+### Testing locally (optional) and upload to dropbox
+
+You can test your curated `AnnData` object after schema application by running with a local installation of cellxgene (`cellxgene launch example.h5ad`). This allows you to preview how your dataset will appear in the cellxgene explorer view within the data portal. Following this optional testing, you need upload to dropbox which is required since datasets cannot be uploaded directly to the portal.
+
+---
+
+## Uploading data to the cellxgene data portal
+
+In general, the cellxgene data portal is oriented around grouping datasets by their association with a particular publication. We refer to all of the datasets associated with a publication as a `collection`. When you go to the homepage of the [cellxgene data portal](#https://cellxgene.cziscience.com/), you will see a list of collections (publications). To explore any of the data associated with these publications, you would click on a particular collection and be greeted by all of the datasets associated with that publication. In this next section, we will cover how to contribute your newly curated data to the cellxgene portal. Briefly, we will cover:
```

Comment:
I clicked on a collection and was not _greeted_ ;-)

Reply:
how rude of them!